### ISSUE TITLE
Ditch npm dependency

### ISSUE THREAD

**Issue by Shmew** [jwosty/Fable.System.IO#5]
I went ahead and wrote up a native implementation of `platform-detect` excluding the out of place parts (the form factor stuff) and added it to [Fable.Extras](https://github.com/Shmew/Fable.Extras). This lets you remove an installation step for users of the library.

So all you need to do is add a reference to that and replace pieces like:

```fsharp
PlatformDetect.os.windows || PlatformDetect.os.uwp
```

with this:

```fsharp
open Fable.Extras.Platform

JSe.Platform.is.windows || JSe.Platform.is.uwp
```

**Comment by jwosty** [comment]
Sweet! I've opened a PR, but there seems to be a problem. More info at #7

**Comment by jwosty** [comment]
Fixed by PR #7
